README.md

```diff
@@ -118,6 +141,7 @@ There are no default mappings, but could easily create them. For example:
 ```
 nnoremap <Leader>rd :PythonCopyReferenceDotted<CR>
 nnoremap <Leader>rp :PythonCopyReferencePytest<CR>
+nnoremap <Leader>ri :PythonCopyReferenceImport<CR>
```

Comment:
I tried this in my local, but `PythonCopyReferenceImport` throws a `not an editor command`. Looks like it needs to be defined as well in the `plugin/python-copy-reference.vim` file as:
```vim
command! -nargs=0 PythonCopyReferenceImport call python_copy_reference#import()
```

Tried adding that line in my local, and it worked after that change.

Reply:
Thanks for spotting again - amended in f56ccf3d76a00fe79f3a85363aeb24345be909ef.

https://github.com/ranelpadon/python-copy-reference.vim/blob/f56ccf3d76a00fe79f3a85363aeb24345be909ef/plugin/python-copy-reference.vim#L17